pom.xml

```diff
@@ -910,6 +910,11 @@
           <artifactId>maven-dependency-plugin</artifactId>
           <version>3.0.1</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-deploy-plugin</artifactId>
+          <version>3.0.0-M1</version>
+        </plugin>
```

Comment:
I've added this because there are two plugins which define a `<skip>true</skip>` for this plugin, but the plugin wasn't defined in `pluginManagement` of the root POM, so maven threw a warning. Adding a specific version to `pluginManagement` makes this warning go away